### ISSUE TITLE
How to log weight, bias, gradient in PyTorch

### ISSUE THREAD

**Issue by ppalantir** [comet-ml/comet-examples#101]
Comet is awesome!

I try to log weight, bias, and gradient in PyTorch by set Experiment as following, but it doesn't work.
```
auto_histogram_gradient_logging=True
auto_histogram_weight_logging=True,
auto_histogram_activation_logging=True
```

Could you give us an example code to log weight, bias, and gradient?

**Comment by Lothiraldan** [comment]
Hi @yxyangxu, thanks for your kind work.

Automatic logging of weight and gradients with Pytorch is not supported yet unfortunately.

I've added a Notebook example how you can log them manually to Comet: https://github.com/comet-ml/comet-examples/blob/master/integrations/model-training/pytorch/notebooks/Histogram_Logging_Pytorch.ipynb

Let us know if that works for you and if you have feedback or additional questions

**Comment by ItamarKanter** [comment]
any update on this issue?
